workflow/github.md

```diff
@@ -1,2 +1,138 @@
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report Bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Add labels:** see [Labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Suggest Enhancements
+
+If you think details could change on our websites you can suggest enhancements by creating a new issue. These are the steps you need follow to suggest enhancements:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Screenshots:** Don’t forget to add screenshots of suggestion designs to explain
+* **Add labels:** see [labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Work on an issue
+You are free to work on an issue if no one seems to work on it. Don’t forget to assign yourself to the issue. Once you finish the work open a [pull-request](#pull-Requests)
+
+## Pull Requests
+
+In this section you will find a description on how to create a pull-request for a Canonical Web-sites project. These are the steps to create a pull-request:
+* Fill the [template](#pull-request-template)
+* Make sure the tests are OK
+* Make sure the [styleguides](#styleguides) are respected
+* Add the [Labels](./labels.md)
+
+### First contribution
+
+If you want to create a pull-request you will need to fork the project you are working on. From your fork, create a branch and work on your issue.
+
+### Styleguides
+
+#### Commits messages
+
+The commit message must describe the feature you want to add to the project.
+
+#### HTML
+
+See [HTML section](../coding/html.md).
+
+#### JS
+
+See [JS section](../coding/js.md)
+
+#### Stylesheets
+
+See [Stylesheets section](../coding/stylesheets.md)
+
+#### Python
+
+See Python section
+
+#### Documentation
+
+See Documentation section
+
+### Merging process
+
+Here are the steps that need to be completed before merging:
+* All [Review labels](./labels.md#review) have to be at +1
+* Your pull request must be approved by at least one person
+* The CI must validate your tests
```

Comment:
Could we say?

> - Any automated tests must have passed

There may not be any tests for some projects - although there should ideally be it's not the submitter's fault if there aren't any.

And I find "continuous integration" a specialised and confusing term, and "CI" is even move obscure.